tests/js/call-missing-first-arg/call-missing-first-arg.js

```diff
@@ -0,0 +1 @@
+foo(, baz);
```

Comment:
I wonder if we should test the cases with and without comments in each of the positions before and after the comma, for example:

```suggestion
foo(, baz);
foo(,);
foo(/*first*/, baz);
foo(,/*second*/ baz);
foo(, baz /* c */);
foo(/* first */, /* second */ baz /* c */);
foo(/* first */,);
foo(, /* second */);
foo(/* first */, /* second */);
```